contracts/BatchExchangeViewer.sol

```diff
@@ -144,6 +145,41 @@ contract BatchExchangeViewer {
         return (elements, nextPageUser, nextPageUserOffset);
     }
 
+    function getEncodedOrdersPaginated(address previousPageUser, uint16 previousPageUserOffset, uint256 pageSize)
```

Comment:
I suppose this function is similar to the incorrectly named one contained in the Exchange contract, but ever so subtly different. Could you comment on any additional subtle differences apart from the array insertion rather than concatenation.

Reply:
It's behaving exactly the same as the incorrectly named on one BatchExchange. Can make a comment to point that out.